SportsHub/SportsHub.Api/Controllers/ArticleController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using SportsHub.AppService.Authentication.Models.DTOs;
+using SportsHub.AppService.Services;
+using System.Text.Json;
+
+namespace SportsHub.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ArticleController : ControllerBase
+    {
+        private readonly IArticleService service;
+        private readonly IMapper mapper;
+
+        public ArticleController(IArticleService service, IMapper mapper)
+        {
+            this.service = service;
```

Comment:
Please do not use `this`. The recommended name for private readonly fields should start from `_`, e.g. _service.
Also, it is not clear from this name what service. It is better to call it `_articleService`.